src/interface/web/app/automations/page.tsx

```diff
@@ -164,68 +164,68 @@ const timestamp = Date.now();
 const suggestedAutomationsMetadata: AutomationsData[] = [
     {
         subject: "Weekly Newsletter",
-        query_to_run:
+        scheduling_request:
```

Comment:
Ah, good find. `scheduling_request` is more like the user query, and `query_to_run` is more like an inferred query. The change in language should help construct more helpful automations.